src/power.jl

```diff
@@ -65,6 +65,15 @@ mutable struct IntervalMatrixPower{T}
         @assert k >= 1 "matrix powers must be positive"
         return new{T}(M, Mᵏ, k)
     end
+
+    function IntervalMatrixPower(M::IntervalMatrix{T}, k::Int) where {T}
```

Comment:
Is there any particular reason to make this an inner constructor? It doesn't need to use `new{T}` if it's an inner constructor? I'm asking because i often have just one inner constructor, the "canonical" one, leaving the others outside (in this case the "canonical" one is in the one in the middle).

Reply:
> Is there any particular reason to make this an inner constructor?

No. I can pull it out.